lib/std/comptime_string_map.zig

```diff
@@ -63,13 +60,42 @@ pub fn ComptimeStringMap(comptime V: type, comptime kvs: var) type {
                 const kv = precomputed.sorted_kvs[i];
                 if (kv.key.len != str.len)
                     return null;
-                if (mem.eql(u8, kv.key, str))
+                if (eql(kv.key, str)) {
                     return kv.value;
+                }
                 i += 1;
                 if (i >= precomputed.sorted_kvs.len)
                     return null;
             }
         }
+
+        pub fn has(str: []const u8) bool {
+            return getWithEqlFn(str, ascii.eql) != null;
+        }
+
+        pub fn get(str: []const u8) ?V {
+            return getWithEqlFn(str, ascii.eql);
+        }
```

Comment:
These can now use `mem.eql` since the len check in `ascii.eql` is redundant.

Reply:
How?
`mem.eql` requires 3 params.

Reply:
Whoops, brain stopped working. See my edit. :smile: